src/bot.ts

```diff
@@ -1325,6 +1328,11 @@ class LemmyBot {
     action: () => Promise<T>;
   }): Promise<T> {
     this.#log(logMessage);
+
+    if (this.#isDryRun) {
+      return Promise.reject();
```

Comment:
There's not a way to call `resolve()` with a body that is going to fulfill arbitrary requests to the lemmy api. This way, by rejecting every time we make it easy for clients to write bots that are resilient to errors.